### ISSUE TITLE
Trapped on Bridge

### ISSUE THREAD

**Issue by Yoshgunn** [Yoshgunn/LightShadowPressRelease#16]
A pathfinding glitch is causing the Protagonist to stall out forever at this point:

![image](https://cloud.githubusercontent.com/assets/4043081/13890882/2577ece6-ed24-11e5-83d3-74992453e4c7.png)

I can't remember where I clicked exactly, but I was certainly going to the other island to hit that button. Then the player froze in place but continued to walk and make footfall noises as usual. This has been going on for a few minutes with no sign of stopping.

**Comment by Jacquies** [comment]
Yeah, this happened to be one as well. I'm looking into it, but I had
trouble reproducing it. It's not happening every time, so we'll be able to
make the video and stuff, but I'll try to figure out why is not working.
